src/pages/components/Overlay.qml

```diff
@@ -228,11 +228,11 @@ PanelBackground {
                 }
                 onShowTabs: {
                     overlayAnimator.showChrome()
-                    // Push the tab index and active page that were current at this moment.
+                    // Push the currently active tab index.
                     // Changing of active tab cannot cause blinking.
+                    webView.grabActivePage()
```

Comment:
```
if (webView.contentItem && !webView.contentItem.loaded) {
    webView.grabActivePage()
}
```
This would reduce flicking with a cost of having tab captures that do not have content from the current position.